demisto_sdk/commands/find_dependencies/find_dependencies.py

```diff
@@ -2009,6 +2031,8 @@ def find_dependencies(
         if not silent_mode:
             # print the found pack dependency results
             click.echo(click.style(f"Found dependencies result for {pack_name} pack:", bold=True))
+            # flatten tuple keys to str to allow parsing to json
+            first_level_dependencies = {str(k): str(v) for k, v in first_level_dependencies.items()}
```

Comment:
is that for display only? do we need to load this JSON back?

Reply:
This is the json output of the function, meaning we will read from this json when wanting to get the dependencies out of the generated json. Since the keys are tuples, dumps fails. Is there a better way abou this?